source/content/guides/secrets/02-basic-concepts.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Pantheon Secrets Guide
+subtitle: Basic Concepts
+description: This section outlines some concepts that are worth knowing about Pantheon Secrets. Gaining familiarity with them will help you to make better use of this feature.
+contributors: [stovak]
+contenttype: [guide]
+innav: [true]
+categories: [secrets]
+cms: [drupal, wordpress]
+audience: [development]
+product: [secrets]
+integration: [--]
+tags: [reference, cli, local, terminus, workflow]
+permalink: docs/guides/secrets/basic-concepts
+reviewed: "2024-05-01"
+---
+<dl>  
+
+<dt>Secret</dt>
+
+<dd>
+
+A key-value pair that should not be exposed to the general public, typically something like a password, API key, or other sensitive information that you should probably not add to version control.
+
+</dd>
+
+<dt>Secret type [^1]</dt>
+
+<dd>
+
+  This is a field on the secret record. It defines the usage for this secret and how it is consumed. Current types are:
+
+  * runtime: this secret will be used to retrieve it in application runtime using API calls to the secret service. This is the recommended type to set if you want your secret to be exposed to the application runtime.
+
+  * env: this secret will be used to set environment variables in the application runtime. This type is currently only supported for Integrated Composer builds and not being exposed as environment variables to the application runtime.
+
+  * composer: this secret type is used for composer authentication to private packages and it is the recommended method for doing so.
+
+</dd>
+
+<dt>Secret Scope</dt>
+
+<dd>
+A secret's scope is the answer to the question "Where is the secret's value available?". Once, set a secret's scope cannot be changed. The secret must be deleted and recreated.
+
+  * `runtime`: this secret will be used to retrieve it in application runtime using API calls to the secret service. This is the recommended type to set if you want your secret to be exposed to the application runtime.
+
+  * `env`: this secret will be used to set environment variables in the application runtime. This type is currently only supported for Integrated Composer builds and not being exposed as environment variables to the application runtime.
+
+  * `composer`: this secret type is used for composer  authentication to private packages and it is the recommended method for doing so.
+
+</dd>
+
+<dt>Owning Entity</dt>
+
+<dd>
+
+* **Organization-owned secrets** -  Organization-owned secrets are available to every site and environment that are associated with the owning organization. A common use-cases is for a CI system and infrastructure that's shared among all sites in an organization.
```

Comment:
We should explicitly note that secrets from supporting organizations are **not** shared with sites that they support.

Reply:
https://github.com/pantheon-systems/documentation/pull/8975/commits/fb9a303efaf79965e1ba2878ae846dc837d4c09b